### PR TITLE
Toggle ai users from being listed among players.

### PR DESCRIPTION
**Pull request by robinlarsson**
I am new to Vue, methods, state and Vuex. So please be skeptical when code reviewing ;)

In this PR I took the liberty of introducing Object.entries in the code base:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

## Default view

AI users are displayed:
image

## Toggle AI users 

to be hidden:
image

### CODE REVIEW

File: games-vue-client/src/components/StartScreen.vue [hunk 1]
```diff
@@ -1,8 +1,21 @@
 <template>
   <v-container fluid>
   <v-row>
+    <v-col cols="12" md="6" lg="4">
+      <v-row>
+        <v-col cols="2" md="2" lg="2">
```
Comment (Zomis) [inline games-vue-client/src/components/StartScreen.vue:6]: Is it necessary to have a v-col in v-row in a v-col in a v-row here? Wouldn't one v-row and the two innermost v-col be enough? Would it be a difference in the view of different devices?

File: games-vue-client/src/components/StartScreen.vue [hunk 2]
```diff
@@ -1,8 +1,21 @@
...
+          <label for="hideAIUsers">Hide AI Users</label>
+        </v-col>
+      </v-row>
+    </v-col>
+  </v-row>
+  <v-row>
+
     <v-col cols="12" md="6" lg="4" v-for="(users, gameType) in lobby" :key="gameType">
-      <LobbyGameType :gameType="gameType" :users="users" :yourPlayer="yourPlayer" />
+      <LobbyGameType :gameType="gameType" :users="usersToDisplay(users)" :yourPlayer="yourPlayer" />
```
Comment (Zomis) [inline games-vue-client/src/components/StartScreen.vue:18]: usersToDisplay should be a computed property and not a method

File: games-vue-client/src/components/lobby/LobbyGameType.vue [hunk 3]
```diff
@@ -61,7 +61,7 @@ export default {
             return supportedGames.games[this.gameType];
         },
         displayName() {
-            return this.supportedGame.displayName ? this.supportedGame.displayName : this.gameType;
+            return this.supportedGame?.displayName || this.gameType;
```
Comment (Zomis) [inline games-vue-client/src/components/lobby/LobbyGameType.vue:64]: Oh, does `?.` work in JavaScript ? Cool! Although I think `this.supportedGame` is never null (or is it?)

File: games-vue-client/src/components/StartScreen.vue (hunk 2 above)
Comment (robinlarsson) [reply games-vue-client/src/components/StartScreen.vue:18]: I fix :)

File: games-vue-client/src/components/lobby/LobbyGameType.vue (hunk 3 above)
Comment (robinlarsson) [reply games-vue-client/src/components/lobby/LobbyGameType.vue:64]: Yeah, it's a nice one with `?.` 

For me, this.supportedGame was null but perhaps it was only and when running against Zomis server.

File: games-vue-client/src/components/StartScreen.vue (hunk 1 above)
Comment (robinlarsson) [reply games-vue-client/src/components/StartScreen.vue:6]: It's not necessary to wrap like I did here, but I like the system of having two or three main columns and arrange items in these main columns into smaller pieces. That gives more control over the white space.

If it's too much for the moment then I will try another solution.

**Pull request merged**